packages/gatsby-theme-carbon/src/components/GlobalSearch/GlobalSearch.js

```diff
@@ -33,9 +33,8 @@ import {
 
 import Menu, { MenuContext } from './Menu';
 
-const MAX_RESULT_LIST_SIZE = 8;
+const MAX_RESULT_LIST_SIZE = 12;
```

Comment:
@eng618 I bumped the default to 12, will also double check with our designers on this, but easy to update later if needed